src/main/java/Entity/Student.java

```diff
@@ -0,0 +1,72 @@
+package Entity;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "student")
+public class Student {
+    @Id
+    @Column
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+    @Column
+    private String firstName;
+    @Column
+    private String lastName;
+    @Column
+    private int age;
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "grooupTitle", referencedColumnName = "title")
```

Comment:
As I understood, you perform join by title, instead of id. Does it mean, that you have Foreig Key as String?